src/Profile/profile.spec.js

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import EditProfile from './EditProfile'
+import "@testing-library/jest-dom/extend-expect";
+import {BrowserRouter as Router} from 'react-router-dom';
```

Comment:
do you use this import?

Reply:
oh i forgot the router part is only for the register page let me remove that

Reply:
also I tried moving the test files to the Tests folder but the tests don't run and I made sure that I was importing properly